src/internal/m365/graph/http_wrapper.go

```diff
@@ -135,6 +169,7 @@ func (pl pipeline) Next(req *http.Request, idx int) (*http.Response, error) {
 func defaultTransport() http.RoundTripper {
 	defaultTransport := http.DefaultTransport.(*http.Transport).Clone()
 	defaultTransport.ForceAttemptHTTP2 = true
+	defaultTransport.IdleConnTimeout = 0 // no limit
```

Comment:
Do we need this particular change given we are retrying http2 errors now? What is the benefit of this change?

I'm a bit concerned if this can have unintended side effects. Not exhaustive, but some scenarios come to mind.

1. Idle connections can become stale, especially if we are doing long running backups. 
2. If idle connections don't get reused for some reason, we may hit max connection limits with graph (assuming there is one).

Reply:
Ah, that was from an earlier test.  I think we can take it out safely.